cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -479,6 +479,11 @@ public static KafkaCluster fromCrd(Kafka kafkaAssembly, KafkaVersion.Lookup vers
             addAll(metricReporterList, configuration.getConfigOption(KAFKA_METRIC_REPORTERS_CONFIG_FIELD).split(","));
         }
 
+        if (kafkaClusterSpec.getReplicas() < 2 && kafkaSpec.getCruiseControl() != null) {
```

Comment:
sorry to be picky, I would invert the conditions in the `if`. Logically you should check the number of replicas only if CC is deployed otherwise it's useless :-)